tendrl/notifier/notification/__init__.py

```diff
@@ -114,12 +107,33 @@ def __init__(self):
             )
             raise ex
 
-    def notify_alert(self, alert):
-        # TODO: Revert to object#load method of loading objects once loading of
-        # nested dicts in objects is fixed in commons.
-        NS.notification_subscriptions = etcd_read(
-            '/notification_settings/config'
-        )
-        if alert is not None:
-            for plugin in NotificationPlugin.plugins:
-                plugin.dispatch_notification(alert)
+    def _run(self):
+        while not self.complete.is_set():
+            try:
+                alerts = get_alerts()
+                # gevent.sleep(30)
```

Comment:
it is required otherwise notifier will read alerting/alerts again and again, so this will used to check alerting/alert only after 30 sec once